control_plane/src/bin/neon_local.rs

```diff
@@ -1052,6 +1054,36 @@ fn get_start_timeout(args: &ArgMatches) -> &Duration {
     humantime_duration.as_ref()
 }
 
+fn get_storage_controller_start_args(args: &ArgMatches) -> NeonStorageControllerStartArgs {
+    let maybe_instance_id = args.get_one::<u8>("instance-id");
+
+    let base_port = args.get_one::<u16>("base-port");
+
+    if maybe_instance_id.is_some() && base_port.is_none() {
+        panic!("storage-controller start specificied instance-id but did not provide base-port");
+    }
+
+    let start_timeout = args
+        .get_one::<humantime::Duration>("start-timeout")
+        .expect("invalid value for start-timeout");
+
+    NeonStorageControllerStartArgs {
+        instance_id: maybe_instance_id.copied().unwrap_or(1),
+        base_port: base_port.copied(),
+        start_timeout: *start_timeout,
+    }
+}
+
+fn get_storage_controller_stop_args(args: &ArgMatches) -> NeonStorageControllerStopArgs {
```

Comment:
nit: we really shouldn't need to name anything `get_` (except get_ref/get_mut alike) as is in the rust style guide https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter but it is quite difficult to get rid of these...

Reply:
Hm, I agree with you, but for me the local style overrides the rust style guide. I can change these if you feel strongly.

Reply:
I think in new code it would be better to follow the style guide which existed even before the local code was added... And as such, the local has no consistent style (even this PR touches code which has or doesn't have `get` prefix).

Reply:
Did some renaming in https://github.com/neondatabase/neon/pull/8587/commits/e288d522b6f8df06f8e17cc92cd340f56ea29d00